app/controllers/application_controller/ci_processing.rb

```diff
@@ -96,7 +96,13 @@ def build_ownership_info
     # need to do this only if 1 vm is selected and miq_group has been set for it
     group = record.miq_group if @ownership_items.length == 1
     @group = group ? group.id.to_s : nil
-    Rbac.filtered(MiqGroup).each { |g| @groups[g.description] = g.id.to_s }
+    miq_groups = if request.parameters[:controller] == 'vm_cloud' &&
+                    request.parameters[:pressed] == 'instance_ownership'
+                   MiqGroup.non_tenant_groups
+                 else
+                   MiqGroup
+                 end
+    Rbac.filtered(miq_groups).each { |g| @groups[g.description] = g.id.to_s }
```

Comment:
Shouldn't the fix here be just
```
-    Rbac.filtered(MiqGroup).each { |g| @groups[g.description] = g.id.to_s }
+    Rbac.filtered(MiqGroup.non_tenant_groups).each { |g| @groups[g.description] = g.id.to_s }
```

?

Otherwise we'd be restricting this logic just to cloud instances, but for e.g. images, we'd still see
the tenant groups in the selection. Or is my thinking here incorrect, @kbrock ?